cloudapi/config.go

```diff
@@ -31,10 +31,12 @@ type Config struct {
 	StopOnError    null.Bool   `json:"stopOnError" envconfig:"K6_CLOUD_STOP_ON_ERROR"`
 	APIVersion     null.Int    `json:"apiVersion" envconfig:"K6_CLOUD_API_VERSION"`
 
-	// TODO: rename the config field to align to the new logic by time series
-	// when the migration from the version 1 is completed.
+	// TODO: remove this when migration from the cloud output v1 will be completed
```

Comment:
At this point, we can move this [at the end of the struct](https://github.com/grafana/k6/blob/ecbe21512901753390e0a12c025f1c9c2cb732f9/cloudapi/config.go#L154) and mark all the _aggregation_ options as deprecated and to be removed after the transition.

Reply:
Resolved with the latest force push